packages/hash/docker/task-executor/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM node:16.14.2-alpine AS builder
+
+WORKDIR /app
+
+COPY packages/hash/task-executor packages/hash/task-executor
+COPY package.json .
+COPY tsconfig.base.json .
+COPY yarn.lock .
+
+RUN --mount=type=cache,mode=0755,target=/yarn-cache \
+  --mount=type=cache,mode=0755,target=/var/cache/apt \
+  --mount=type=cache,mode=0755,target=/var/lib/apt \
+  yarn workspace @hashintel/task-executor install --cache-folder /yarn-cache \
```

If the prefix is adopted, then it should be added here
```suggestion
  yarn workspace @hashintel/hash-task-executor install --cache-folder /yarn-cache \
```